src/main/java/org/opensrp/repository/postgres/mapper/custom/CustomLocationMetadataMapper.java

```diff
@@ -47,4 +47,7 @@
 	Location findByIdAndVersion(@Param("id") String id, @Param("geometry") boolean returnGeometry, @Param("version") int version);
 
 	List<LocationDetail> selectLocationWithDescendants(@Param("locationId") String locationId, @Param("tags") boolean returnTags);
+
+	List<String> selectChildrenIds(@Param("geometry") boolean returnGeometry,
```

Comment:
since we are returning Ids, `@Param("geometry") boolean returnGeometry` is not required

Reply:
Removed.